ui/src/views/AutogenView.vue

```diff
@@ -983,13 +983,13 @@ export default {
               continue
             }
             if (!input === undefined || input === null ||
-              (input === '' && !['updateStoragePool', 'updateHost', 'updatePhysicalNetwork'].includes(action.api))) {
+              (input === '' && !['updateStoragePool', 'updateHost', 'updatePhysicalNetwork', 'updateDiskOffering', 'updateNetworkOffering'].includes(action.api))) {
               if (param.type === 'boolean') {
                 params[key] = false
               }
               break
             }
-            if (!input) {
+            if (!input && !['tags'].includes(key)) {
```

Comment:
I think it should be `key !== 'tags'`

Reply:
@utchoang should we re-look this behaviour of skipping params that do not have a value in API request.
It should be already handled by the `required` property of the parameter and there can be some parameters that may allow passing empty text.
`!['tags'].includes(key)` should work same as `key !== 'tags`?

Reply:
@utchoang I left it like this `!['tags'].includes(key)` for future fixes/enhancements if somebody wants to add parameters that could be an empty text. If this is not acceptable, I could change it with your suggestion